absl/base/internal/unscaledcycleclock.cc

```diff
@@ -57,11 +62,34 @@ double UnscaledCycleClock::Frequency() {
 #elif defined(__powerpc__) || defined(__ppc__)
 
 int64_t UnscaledCycleClock::Now() {
+#ifdef __GLIBC__
   return __ppc_get_timebase();
+#elif defined(__FreeBSD__)
+  union { long long complete; unsigned int part[2]; } ticks;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Integer_Types

This is a case where you should use fixed-width integer types like `int64_t`.